apps/site/next.config.js

```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: false,
+  images: {
+    domains: ['res.cloudinary.com'],
```

Comment:
Vercel using this domain?

Reply:
The images from the old website are hosted on that